test/specs/elements/Segment/Segments-test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {Segment, Segments} from 'stardust';
+
+describe('Segments', () => {
+  it('should render children', () => {
+    const [segmentOne, segmentTwo] = render(
+      <Segments>
+        <Segment>Top</Segment>
+        <Segment>Bottom</Segment>
+      </Segments>
+    ).scryClass('sd-segment');
+
+    segmentOne
+      .textContent
+      .should.equal('Top');
+    segmentTwo
+      .textContent
+      .should.equal('Bottom');
+  });
+
+  it('renders expected number of children', () => {
+    const [component] = render(
+      <Segments>
+        <Segment>Top</Segment>
+        <Segment>Middle</Segment>
+        <Segment>Bottom</Segment>
+      </Segments>
+    ).scryClass('sd-segments');
+
+    expect(
+      component.querySelectorAll('.sd-segment').length
+    ).to.equal(3);
+  });
```

Comment:
Since we are testing how many `.sd-segment` elements there are, we can forgo accessing them through the `.sd-segments` element and directly test the count:

```jsx
it('renders expected number of children', () => {
  render(
    <Segments>
      <Segment>Top</Segment>
      <Segment>Middle</Segment>
      <Segment>Bottom</Segment>
    </Segments>
  )
    .scryClass('sd-segment')
    .should.have.a.lengthOf(3);
});
```
>Makes use of chai [length/lengthOf](http://chaijs.com/api/bdd/#length-section) assertion.